### PR TITLE
Various minor fixes

### PR DESCRIPTION
**Pull request by aosolis**
- tslint errors from typings
- set max url length to 25000
- make server:start gulp task work

### CODE REVIEW

File: deploy.cmd [hunk 1]
```diff
@@ -96,6 +96,7 @@ IF /I "%IN_PLACE_DEPLOYMENT%" NEQ "1" (
 
 :: 2. Select node version
 call :SelectNodeVersion
+call :ExecuteCmd !NPM_CMD! config set scripts-prepend-node-path true
```
Comment (jotrick) [inline deploy.cmd]: For my benefit - what are these changes doing in the deploy.cmd?

File: src/apis/ValidateAADToken.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ export class ValidateAADToken {
             // Decode token and get signing key
             const encodedToken = authHeaderMatch[1];
             const decodedToken = jwt.decode(encodedToken, { complete: true });
-            msaOpenIdMetadata.getKey(decodedToken.header.kid, (key) => {
+            msaOpenIdMetadata.getKey(decodedToken["header"].kid, (key) => {
```
Comment (jotrick) [inline src/apis/ValidateAADToken.ts]: Why this change and the below?

Comment (aosolis) [reply src/apis/ValidateAADToken.ts]: The jsonwebtoken typing defines decode() as returning { [key: string]: any }.
